### PR TITLE
Feature/sdl rc merge to extended policy functionality

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[submodule "sdl"]
```
Comment (istoilovgithub) [inline .gitmodules]: @LuxoftAKutsan Most probable this one was committed by mistake, please remove.

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Aleksandr Kutsan <akutsan@luxoft.com> akutsan <akutsan@akutsan>
```
Comment (istoilovgithub) [inline .mailmap]: @LuxoftAKutsan The same story as above.

File: .gitmodules (hunk 1 above)
Comment (LuxoftAKutsan) [inline .gitmodules]: @istoilovgithub 8f69212

File: .mailmap (hunk 2 above)
Comment (LuxoftAKutsan) [inline .mailmap]: @istoilovgithub fixed in  8f69212

**Pull request merged**